src/renderer/components/inputs/props.ts

```diff
@@ -1,10 +1,18 @@
-import { InputData, InputId, InputSchemaValue, SchemaId } from '../../../common/common-types';
+import {
+    InputData,
+    InputId,
+    InputSchemaValue,
+    InputSize,
+    SchemaId,
+    Size,
+} from '../../../common/common-types';
 import { Type } from '../../../common/types/types';
 
 export interface InputProps {
     readonly id: string;
     readonly inputId: InputId;
     readonly inputData: InputData;
+    readonly inputSize?: InputSize;
```

Comment:
Is this even necessary? Why would any input use `inputSize` when we have `useInputSize`?

Reply:
I just copied how inputData/useInputData was, so you tell me lol

Reply:
We inputs even use `inputData`....

Reply:
Nope, they don't. Huh, well that's the next thing we can remove.